### ISSUE TITLE
Configurable text (speech) speed

### ISSUE THREAD

**Issue by balloonpopper** [godot-escoria/escoria-issues#194]
**Is your feature request related to a problem? Please describe.**
The speech in the demo plays too slowly

**Describe the solution you'd like**
Configurable speech speed (with setting in the sample UI to adjust it)

**Comment by dploeger** [comment]
This is currently possible for the developer: https://github.com/godot-escoria/escoria-dialog-simple/tree/develop

Would you like to have it configurable by the player?

**Comment by balloonpopper** [comment]
Yes, configurable by the player. I read fast and get very bored very quickly when game text isn't really fast.

**Comment by dploeger** [comment]
Okay. Then I'll postpone it to future if that's okay. (Basically, it's the responsibility of the game dev who creates their own UI to implement a feature like this, but we want to add accessibility settings anyway in future)
